e2e/app/page/notebook-page.ts

```diff
@@ -51,7 +51,7 @@ export default class NotebookPage extends AuthenticatedPage {
       console.log(`Reloading "${this.documentTitle}" because cannot find the Run button`);
       await this.page.reload({waitUntil: ['networkidle0', 'load']});
     }
-    await this.waitForKernelIdle(120000);
+    await this.waitForKernelIdle(180000); // 3 minutes
```

Comment:
Fix a timeout failure:

https://app.circleci.com/pipelines/github/all-of-us/workbench/6396/workflows/3df44fa7-e8b5-448c-9c67-3ab504e74f1f/jobs/113642/steps

```
FAIL tests/notebook/notebook-python3.spec.ts (290.995 s)
  Jupyter notebook tests in Python language
    ✕ Run code from file (290223 ms)

  ● Jupyter notebook tests in Python language › Run code from file

    waitForKernelIdle encountered TimeoutError: waiting for selector `#MathJax_Message` to be hidden failed: timeout 120000ms exceeded

      126 |     } catch (e) {
      127 |       console.error(`Notebook kernel is: ${await this.kernelStatus()}`);
    > 128 |       throw new Error(`waitForKernelIdle encountered ${e}`);
          |             ^
      129 |     }
      130 |   }
      131 | 

      at NotebookPage.waitForKernelIdle (app/page/notebook-page.ts:128:13)
      at NotebookPage.isLoaded (app/page/notebook-page.ts:54:5)
      at NotebookPage.waitForLoad (app/page/authenticated-page.ts:35:5)
      at WorkspaceAnalysisPage.createNotebook (app/page/workspace-analysis-page.ts:77:5)
      at Object.<anonymous> (tests/notebook/notebook-python3.spec.ts:29:22)
```